### PR TITLE
added missing style tags and improved structure/spacing

### PR DESCRIPTION
**Pull request by voisardf**
Mostly styling and improving. thanks for a quick look

### CODE REVIEW

**Comment by maltaesousa** [comment]
@voisardf reviews on large diffs like these are time consuming and hard to do. Is there any place where this mapfile can be checked like a prepub instance with an url example.

Also when improving mapfile syntax like this, consider using this library : https://mappyfile.readthedocs.io/en/latest/client.html#validate

It helps to do syntax check :)

File: mapserver/crdppf.map.mako [hunk 1]
```diff
@@ -211,81 +211,81 @@ LAYER
...
-        STYLE
-            PATTERN
-                10 5
-            END
-            COLOR 170 168 0
-            WIDTH 1.5
+  END #CLASS
+  CLASS
+    NAME "Alignement obligatoire"
+    EXPRESSION /7113/
```
Comment (kalbermattenm) [inline mapserver/crdppf.map.mako:217]: All classes are defined as regular expressions. Is that right?

File: mapserver/crdppf.map.mako [hunk 2]
```diff
@@ -719,170 +719,170 @@ LAYER
...
+        GAP 20
+    END
+    STYLE
+        OUTLINECOLOR  64 64 64
+        WIDTH 1
+    END #STYLE
+  END #CLASS
+  CLASS
+    NAME "Zone réservée cantonale"
+    EXPRESSION ([codegenre] = 5912 AND '[teneur]' eq 'Zone réservée cantonale')
```
Comment (kalbermattenm) [inline mapserver/crdppf.map.mako:750]: Isn't it to mix full expression definitions with classitem definitions (line 764)? Wouldn't it be better to have everything in full expression in this case?

File: mapserver/crdppf.map.mako [hunk 3]
```diff
@@ -1390,8 +1398,10 @@ LAYER
   OPACITY 60
   CLASS
     NAME "Alignements des install. aéroportuaires"
+    STYLE
     COLOR 0 0 180
```
Comment (kalbermattenm) [inline mapserver/crdppf.map.mako:1402]:
```suggestion
      COLOR 0 0 180
```

File: mapserver/crdppf.map.mako [hunk 4]
```diff
@@ -1441,39 +1451,51 @@ LAYER
   CLASS
     NAME "Pollué, investigation nécessaire"
     EXPRESSION /9903/
-    COLOR 0 0 255
-    OUTLINECOLOR 0 0 0
+    STYLE
+        COLOR 0 0 255
```
Comment (kalbermattenm) [inline mapserver/crdppf.map.mako:1455]: Is your indent 2 spaces or 4 spaces long?

Comment (kalbermattenm) [reply mapserver/crdppf.map.mako:1455]: Same down under, in the `STYLE` you mostly used 4 spaces and 2 spaces elsewhere (but this is not of top importance...)

**Review by kalbermattenm — APPROVED** [review]

**Comment by voisardf** [comment]
@maltaesousa I know!
I put the branch on prepub: https://sitn.ne.ch/crdppf_prepub

oh nice library! Didn't know about that possibility.

File: mapserver/crdppf.map.mako (hunk 4 above)
Comment (voisardf) [reply mapserver/crdppf.map.mako:1455]: it's 4 now in VS Code. But there are still some old copy-paste left

File: mapserver/crdppf.map.mako (hunk 2 above)
Comment (voisardf) [reply mapserver/crdppf.map.mako:750]: classes became more complexe with the new amenagement and dangers naturels structure, thus the full expressions.
I could clean up and update all classes to full expression and latest mapfile definitions. But that would be even more time consuming. So I hesitate...

Any thoughts/recommandations?

Comment (kalbermattenm) [reply mapserver/crdppf.map.mako:750]: Nope, it is just strange to see that mix... but there is no good/rapid solution with such a long mapfile...

Comment (voisardf) [reply mapserver/crdppf.map.mako:750]: ok, thanks. Then I will gradually adapt all layers for the new version

**Comment by voisardf** [comment]
now it should be clean! thanks for your hints 👍

**Pull request merged**